glossary.md

```diff
@@ -1,16 +1,24 @@
-# Glossary for Alternate and Alternative Encodings
+# Glossary for Alternative Encodings
 
-## Alternate Encoding
+## INSPIRE Alternative Encoding Rule
 
-An Alternate Encoding may be used to encode an INSPIRE data set instead of using the Default Encoding, since, for that data set, all requirements of the Implementing Rule can be met using the Alternate Encoding.
+alternative encoding rule
```

Comment:
I think we need to highlight synonyms more clearly, maybe as notes rather than directly under the term, i,e.

NOTE Synonyms: alternative encoding rule 

Or we can put it in square brackets... (but then it should be explained at the beginning of the document)

Reply:
@michellutz @thorsten-reitz 
Putting the synonyms (admitted terms) under the term is the way it is suggested in ISO 10241-1 (Annex A, informative), and is the way it is done in other ISO standards.

So if those guidelines are followed, the structure and formatting should be something like this:
**preferred term**
admitted term 1
admitted term 2

definition

EXAMPLE an example

NOTE 1 A note.

NOTE 2 Another note.

[SOURCE a source]

Or maybe like this (using the `<dl>`, `<dt>` and `<dd>` HTML tags, see code, see also e.g. https://inspire-eu-rdf.github.io/inspire-rdf-guidelines/#_terms_and_abbreviations for examples):
 <dl>
  <dt>preferred term</dt>
  <dd>
<p>admitted term 1<br />
admitted term 2
</p>
<p>definition</p>
<p>EXAMPLE an example</p>
<p>NOTE 1 A note.</p>
<p>NOTE 2 Another note.</p>
<p>[SOURCE a source]</p>
</dd>
</dl> 

How about following those guidelines, with an explanatory comment on how glossary entries are structured?

Reply:
+1 for Heidi's last suggestion

Reply:
There is no nice way of achieving the indent in markdown. I have made the other changes though.